test/phlex/view/tags.rb

```diff
@@ -122,4 +122,40 @@
 			end
 		end
 	end
+
+	describe "Elements deprecation" do
+		deprecated_methods = %i[param]
+
+		deprecated_methods.each do |method_name|
```

Comment:
I'm not sure if worths to test non deprecated elements. I would be something like:
```ruby
[
	Phlex::HTML::StandardElements.registered_elements,
	Phlex::HTML::VoidElements.registered_elements
].each do |element_group|
	element_group.each do |method_name, tag|
    if deprecated_methods.include?(method_name)
      # tests for method_name and _method_name
      ... warns about deprecation
    else
      # tests for method_name and _method_name
      ... doesn't warn about deprecation
    end
  end
end
```